src/todo.js

```diff
@@ -0,0 +1,142 @@
+let tasks = [{
+  description: 'Task 1',
+  completed: false,
+  index: 1,
+},
+{
+  description: 'Task 2',
+  completed: true,
+  index: 2,
+},
+// Add more tasks with different index values
+];
+
+function updateIndexes() {
+  tasks.forEach((task, index) => {
+    task.index = index + 1;
+  });
+}
+
+export function deleteTask(todoItem) {
+  todoItem.remove();
+}
+
+export function editTask(todoItem) {
+  const taskElement = todoItem.querySelector('.task');
+  const taskText = taskElement.textContent;
```

Comment:
- [ ] Also, when we are editing the Task, we are not updating our local storage. Kindly fix it 👍